[ADD] rental: new product types page (#17997)
odoo/documentation: content/applications/sales/rental/configure_products/product_types.rst

====================
Rental product types
====================

> Odoo allows for two product types when configuring a rental product: Goods or Services.

Reviewer: is there a reason why 'Goods' and 'Services' are italicized in the next sentence but not here?

Author: I think I wanted to start italicizing them when I introduced the descriptions of each term.

A rental *Goods* product is a physical item, such as a computer, vehicle, bike, camera, or clothing.

> Goods are removed from the company’s stock, delivered to the customer, and then returned.

Reviewer: ```suggestion
A rental *Goods* product is a physical item (e.g., a computer, vehicle, clothing) that is removed from the company’s stock, delivered to the customer, and then returned.
```

Reviewer: suggestion for brevity. also, i think having two or three examples of physical items suffices.

A rental *Service* product can be labor or something non-physical, such as installation,
photography, or a deposit. It can also be a physical item that never leaves the company’s stock,

> such as a hotel room, work station, or storage unit. The following sections detail basic settings

Reviewer: ```suggestion
A rental *Service* product can be a service or an intangible good (e.g., installation,
photography, a deposit), or it can be a physical item that never leaves the company’s stock (e.g., a hotel room, work station, or storage unit). The following sections detail basic settings
```

Reviewer: feel free to just go with two sentences here instead of the conjunction in the middle - i'm just overly obsessed with condensing. 

i would, however, suggest changing the "something non-physical" to "intangible good" as more specific and widely-known term for what you're describing. also "labor" could be changed to "service".

> and app integration configurations for Goods and Services.

Reviewer: ```suggestion
such as a hotel room, work station, or storage unit. 

The following sections detail basic settings
and app integration configurations for Goods and Services.
```

Reviewer: just a suggestion to separate definition from the statement of intent

Setting
=======

.. important::

>    The Configuration menu is only available if the **Inventory** app or **eCommerce** module is

Reviewer: ```suggestion
   The *Configuration* menu is only available if the **Inventory** app or **eCommerce** module is
```

Reviewer: just to clarify: does this mean that the whole configuration menu doesn't exist if these apps aren't installed? or just that their respective settings do not appear?

Author: Yup, the Configuration menu isn't available at all if Inventory or eCommerce isn't installed.

   installed.

   - For the :guilabel:`Rental Transfers` setting, the **Inventory** app must be installed.
   - For the :guilabel:`Unavailability days` setting, the **eCommerce** module must be installed.

To configure default settings on rental products, navigate to :menuselection:`Rental app -->
Configuration --> Settings`.

> In the *Rental* section, the :guilabel:`Rental Transfers` checkbox enables automatic creation of

Reviewer: ```suggestion
In the *Rental* section, the :guilabel:`Rental Transfers` setting allows Odoo to automatically create
```

delivery and return receipts for a rental product.

.. note::
   The **Inventory** app automatically creates an internal default location once the
   :guilabel:`Rental Transfers` feature is enabled. Odoo uses the new default location,
   `Customer/Rental`, to track products during the rental period (moving them from Stock to
   Customer/Rental upon rental, and back upon return). Do not modify this location avoid corrupting
   inventory tracking.

> If the **eCommerce** module is installed, the *Rent Online* section is available to configure. The

Reviewer: ```suggestion
Next, the *Rent Online* section is available to configure. The
```

Reviewer: This conditional "if" here is established already at the beginning in the admonition. I would suggest removing this "if" here if you decide to go with the admonition block. Less repetition of info.

:guilabel:`Unavailability days` limit what days of the week customers can book or return rental
products online.

Click :guilabel:`Save` to apply the changes.

.. image:: product_types/rental-settings-with-inventory-ecommerce.png
   :alt: Settings page of the Rental app with the Inventory and eCommernce apps installed.

App integration configuration
=============================

> The Rental app relies on additional Odoo app integration to expand its settings and product

Reviewer: ```suggestion
The **Rental** app relies on additional Odoo app integrations to expand its settings and product
```

configurations. The following apps are essential for workflow efficiency and automation when
creating a product and rental order:

> - Sales: Enables online payments and signatures within the Rental app.

Reviewer: ```suggestion
- **Sales**: Enables online payments and signatures within the **Rental** app.
```

- Sign: Allows uploading and customization of various rental and service agreements. These documents

>   are used to facilitate the Request Signature feature.

Reviewer: ```suggestion
- **Sign**: Allows uploading and customization of various rental and service agreements. These documents
  are used to facilitate the *Request Signature* feature.
```

Reviewer: is this 'request signature' feature specific to rental or sign app? is there another doc to link to?

Author: This feature is tied to the Rental app--but it requires the Sign app to be installed for it to be available.

Author: I added the Sign link in the See also section

> - Planning: Integrates with the Rental app to automatically match service products with employees or

Reviewer: ```suggestion
- **Planning**: Integrates with the **Rental** app to automatically match service products with employees or
```

  materials based on availability. This setting is configured on the product form.

> - Project: Enables automatic creation of a project, a task, or both when a rental quote that

Reviewer: ```suggestion
- **Project**: Enables automatic creation of a project, a task, or both when a rental quote that
```

  includes the configured product is confirmed. This setting is configured on the product form.

> - eCommerce: Allows product configuration for the online shop. This setting is configured on the

Reviewer: ```suggestion
- **eCommerce**: Allows product configuration for the online shop. This setting is configured on the
```

  product form.

.. seealso::

   - Quotation templates
   - Online payment order confirmation
   - Online signatures for order confirmation
   - PDF Quote Builder
   - Planning
   - Sign
   - Project

>    - eCommerce

Reviewer: Links to these docs are missing

Search for rental products
==========================

> To view all products available for rent in the database, navigate to Rentals app ‣ Products. By

Reviewer: ```suggestion
To view all products available for rent in the database, navigate to :menuselection:`Rentals app --> Products`. By
```

> default, the Rental filter appears in the search bar, and the view is Kanban. Click the search bar

Reviewer: ```suggestion
default, the *Rental* filter appears in the search bar, and the view is Kanban. Click the search bar
```

and from the preset filters, select Goods, Services, or both. All the selected options appear as

> Kanban cards. For Goods, the card displays the name, rental rate, and amount on hand. For Services,

Reviewer: the "amount on hand" depends on if the "track inventory" option is enabled on the product form... if i'm not mistaken. if so, might be worth clarifying that with a few words.
also, please check if any other fields appear based on configurations.

> the card displays the name, the number of variants if configured, and the rental price.

Reviewer: So this section mentions the Kanban view, what about the List view? Might be worth briefly explaining what users see.

> the card displays the name, the number of variants if configured, and the rental price.

Reviewer: ```suggestion
and from the preset filters, select Goods, Services, or both. For Goods, the card displays the name, rental rate, and amount on hand. For Services,
the card displays the name, the number of variants if configured, and the rental price.
```

.. seealso::
   - Configure a service product

>    - Configure a physical product

Reviewer: Links to these docs are missing

====================
Rental product types
====================

Odoo allows for two :ref:`product types <inventory/product_management/good-or-services>` when
configuring a rental product: Goods or Services.

*Goods*: Includes physical items. Goods are removed from the company’s stock, delivered to the
customer, and then returned.

> *Services*: Includes labor, intangible good, or a physical item that never leaves the company’s

Reviewer 2: ```suggestion
*Services*: Includes labor, an intangible good, or a physical item that never leaves the company’s
```

The following sections detail basic settings and app integration configurations for goods and
services.

Configuration
=============

.. important::
   The *Configuration* menu is only available if the **Inventory** app or **eCommerce** module is
   installed.

   - For the :guilabel:`Rental Transfers` setting, the **Inventory** app must be installed.
   - For the :guilabel:`Unavailability days` setting, the **eCommerce** module must be installed.

To configure default settings on rental products, navigate to :menuselection:`Rental app -->
Configuration --> Settings`.

.. image:: product_types/rental-settings-with-inventory-ecommerce.png
   :alt: Settings page of the Rental app with the Inventory and eCommerce apps installed.

In the *Rental* section, the :guilabel:`Rental Transfers` checkbox enables automatic creation of
delivery and return receipts for a rental product.

.. note::
   The **Inventory** app automatically creates an internal default location once the
   :guilabel:`Rental Transfers` feature is enabled. Odoo uses the new default location,
   `Customer/Rental`, to track products during the rental period (moving them from Stock to

>    Customer/Rental upon rental, and back upon return). **Do not** modify this location avoid

Reviewer 2: ```suggestion
   Customer/Rental upon rental, and back upon return). **Do not** modify this location to avoid
```
